packages/playground/cli/src/cli.ts

```diff
@@ -215,7 +215,6 @@ async function run() {
 					php: args.php as SupportedPHPVersion,
 					wp: args.wp,
 				},
-				login: args.login,
```

Comment:
What happened here? Why remove this?

Reply:
It was causing errors and it's not supported anymore since we changed the way login works. Should I open an issue to add login support?

We have an [option to login users with CLI](https://github.com/WordPress/wordpress-playground/blob/bef1be234a5d11b322292380303e5716db3e5108/packages/playground/wordpress/src/index.ts#L69), but it requires a specific URL parameter (`playground_force_auto_login_as_user`), we could print this in the CLI output and enable users to login that way.

Reply:
Why couldn't login work the same way as on Playground web?

Reply:
PR to fix login https://github.com/WordPress/wordpress-playground/pull/1985